packages/vite/src/node/preview.ts

```diff
@@ -28,9 +28,12 @@ export async function preview(
   )
 
   // cors
-  const { cors } = config.server
-  if (cors !== false) {
-    app.use(corsMiddleware(typeof cors === 'boolean' ? {} : cors))
+  if (config.server.cors !== false) {
+    app.use(
+      corsMiddleware(
+        typeof config.server.cors === 'boolean' ? {} : config.server.cors
+      )
+    )
```

Comment:
> The destructuring-like pattern is only used to avoid duplication (in the case of httpServer = server.httpServer, or a very long identifier that is used many times (in the case of info = server.config.logger.info).

Here `config.server.cors` is used 3 times. It's okay to use a new variable for a shorter name.

---

I'm not requesting a change.

I'm against the overall refactoring because it doesn't improve the project's maintainability.

Reply:
ok,thx.I will close this pull request